server/Models/Items.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * server/Models/Items.js
+ *
+ * @routes /api/inventories
+ * @todos
+ *
+ */
+const MONGOOSE = require('mongoose');
+
+const SCHEMA = MONGOOSE.Schema;
+
+const ITEM = new SCHEMA({
```

Comment:
Eventually I would like for this to be a function with the parameter as an array provided by the user, thus creating their own collection with user-defined key-values